tests/test_volume.py

```diff
@@ -120,6 +130,88 @@ def testProject(self):
             #  centered along the rotation axis for multiples of pi/2.
             self.assertTrue(np.allclose(vol_along_axis, prj_along_axis))
 
+    def testRotate(self):
+        # Create a stack of rotations to test. We will rotate by multiples of pi/2 from each axis.
+        rot_mat = np.empty((12, 3, 3), dtype=self.dtype)
+        axes = ["x", "y", "z"]
+        angles = [0, np.pi / 2, np.pi, 3 * np.pi / 2]
+        pairs = list(product(axes, angles))
+        for k in range(len(pairs)):
```

Comment:
Here I probably would change, maybe:

```
for k, pair in product(axes,angles):
            rot_mat[k] = Rotation.from_euler(*pair).as_matrix()
```

or 
```
for k, (ax,ang) in product(axes,angles):
            rot_mat[k] = Rotation.from_euler(ax, ang).as_matrix()
```

Reply:
That's much cleaner. Thanks. Done